src/mugs.js

```diff
@@ -1,3 +1,37 @@
+/**
+ * A mug is a question, containing data, bind, and control elements.
+ *
+ * Main Properties
+ *  nodeID
+ *  label
```

Comment:
I 100% agree javarosa should be kept as a plugin. I think the benefits of having it in a contained area outweigh anything else.

What I meant by my original comment was that things like `label` and `hintLabel` could be removed from src/mugs.js, and I assumed that there are probably some other things that could be removed as well, although I haven't checked.

That is unless we can see a future where vellum can be run without javarosa where a label would be needed